auto_round/export/export_to_itrex/export.py

```diff
@@ -170,17 +170,23 @@ def pack_model(
         q_config = weight_config
     for k, v in q_config.items():
         logger.info(f"Packing {k}")
-        if "float" in v["data_type"]:
+        try:
+            if "float" in v["data_type"]:
+                continue
+            dtype = v["data_type"]
+            num_bits = v["bits"]
+            group_size = v["group_size"]
+            sym = v["sym"]
+            scale_dtype = v["scale_dtype"]
+            scale, zp = v["scale"], v["zp"]
+        except:
```

Comment:
why using try except,   having no scale info in the dict could detect this issue